README.md

```diff
@@ -42,12 +42,12 @@ export default {
 
 To customize formatting behavior, see the [Configuration](#configuration) section below.
 
-## Using in VS Code
+## Using in VS Code with the Prettier extension
```

Comment:
```suggestion
## Formatting with the VS Code Prettier extension directly
```
I think something like this can help prime the reader for the fact that you're going to be talking about a DIFFERENT, other extension

Reply:
That's fine by me!